### ISSUE TITLE
Model.init() give type error when include pressure="lithostatic"

### ISSUE THREAD

**Issue by HanyMKhalil** [underworldcode/underworld2#646]
Dear all,

I am trying to run a convergence code however I got the following error when i define the pressure of the base to be lithostatic:

    Model.init_model(temperature="steady-state", pressure="lithostatic")
TypeError: init_model() got an unexpected keyword argument 'pressure'

**Comment by HanyMKhalil** [comment]
can any one help me how to create lithostatic pressure on bottom? I am trying to run the convergence model with air, sticky air and crust layer only on the base
the line does not work

Model.init_model(temperature="steady-state", pressure="lithostatic")
TypeError: init_model() got an unexpected keyword argument 'pressure'

**Comment by bknight1** [comment]
what version of UW/UWGeo are you using?

I've just tried v2.14 and it works as expected `Model.init_model(temperature="steady-state", pressure="lithostatic")`

Cheers

**Comment by HanyMKhalil** [comment]
Hey Ben I am using the one installed on Gadi already, and gave me this error.

**Comment by HanyMKhalil** [comment]
but the result are not the same if I removed thew pressure word, as it works fine on the version on Monarch but on Gadi is not working and the results are deferent, so I suppose I need the code to run with lithospheric pressure.

**Comment by bknight1** [comment]
Yeah that's because they'll be different versions of UW, you'll have to update the version on gadi

**Comment by HanyMKhalil** [comment]
is there any work around this problem? like the code on Gadi has to have somewhere where you can recognise pressure as lithostatic? I am not sure I will be able to update the version on Gadi easily I do not wanna miss stuff up.
if I left it blank will it be lithostatic by default?

**Comment by bknight1** [comment]
I believe it defaults to lithostatic, so it should be fine
